iothub_client/src/iothub_client_ll_uploadtoblob.c

```diff
@@ -774,10 +834,10 @@ IOTHUB_CLIENT_RESULT IoTHubClient_LL_UploadToBlob_Impl(IOTHUB_CLIENT_LL_UPLOADTO
     if (
         (handle == NULL) ||
         (destinationFileName == NULL) ||
-        ((source == NULL) && (size > 0))
+        (getDataCallback == NULL)
         )
     {
-        LogError("invalid argument detected handle=%p destinationFileName=%p source=%p size=%zu", handle, destinationFileName, source, size);
+        LogError("invalid argument detected handle=%p destinationFileName=%p getDataCallback=%p context=%zu", handle, destinationFileName, getDataCallback, context);
```

Comment:
Please do not print the context. Not needed.

Reply:
Done